.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,79 @@
+name: Feature Request Form
+description: Request new or improved functionality
+title: "[FEA]: "
+labels: ["feature request"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
+
+  - type: dropdown
+    id: new_or_improvement
+    attributes:
+      label: Is this a new feature, or an improvement?
+      options:
+        - New Feature
+        - Improvement
+    validations:
+      required: true
+
+  - type: dropdown
+    id: criticality
+    attributes:
+      label: How would you describe the priority of this feature request
+      options:
+        - Critical (currently preventing usage)
+        - High
+        - Medium
+        - Low (would be nice)
+    validations:
+      required: true
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: Please provide a clear description of problem this feature solves
+      description: Real usage examples are especially helpful, non-code.
+      placeholder: I want Morpheus to do _____, because I need to _____.
+    validations:
+      required: true
+
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe your ideal solution
+      description: Please describe the functionality you would like added.
+      placeholder: >
+        A new function that takes in the information in this form, and triages the issue
+
+        def feature_request(form_info):
+            parse(form_info)
+            return triage_outcome
+    validations:
+      required: true
+
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Describe any alternatives you have considered
+      description: List any other libraries, or approaches you have looked at or tried.
+      placeholder: I have looked at library xyz and qrs, but they do not offer GPU accleration
+
+  - type: textarea
+    id: misc
+    attributes:
+      label: Additional context
+      description: Add any other context, code examples, or references to existing implementations about the feature request here.
+
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/nv-morpheus/Morpheus/blob/main/CODE_OF_CONDUCT.md)
+      options:
+        - label: I agree to follow this project's Code of Conduct
+          required: true
+        - label: I have searched the [open issues](https://github.com/nv-morpheus/morpheus/issues) and have found no duplicates for this feature request
+          required: true
```

Comment:
We should add this to bugs as well

Reply:
Added it to each of the forms, and changed to filter to the correct issue type as well.